packages/compartment-mapper/test/fixtures-optional/node_modules/optional-cjs/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports.tryOptionalDeps = () => {
+  const results = { exports: {}, errors: {} };
+  try {
+    results.exports.alpha = require('alpha');
+  } catch (err) {
+    results.errors.alpha = err;
+  }
+  try {
+    results.exports.beta = require('beta');
+  } catch (err) {
+    results.errors.beta = err;
+  }
+  try {
+    results.exports.one = require('missing-one');
+  } catch (err) {
+    results.errors.one = err;
+  }
+  try {
+    results.exports.two = require('missing-two');
+  } catch (err) {
+    results.errors.two = err;
+  }
+  return results
+}
```

Comment:
Such Windows file without newline on the end.

Reply:
tell it to the linter rules :tongue: